src/HomePage/HomePage.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { View, Image } from 'react-native';
+import { Button } from "react-native-paper";
+import { getAllRooms } from '../../apiCalls';
+import Overview from '../Overview/Overview'
+
+const HomePage = () => {
+  const handlePress = () => {
+
+  }
```

Comment:
I think this is where we will handle the API call to fetch all the Rooms.  Not sure about how the navigation works either but we may want to handle that here too in the future.